### PR TITLE
FEATURE: BEND-19 Get Photo by ID endpoint

### PR DESCRIPTION
**Pull request by octapf**

### CODE REVIEW

**Comment by whitehatdevv** [comment]
@octapf check the new name I have just put, it's recommended to set the right name

File: src/http/actions/Account.ts [hunk 1]
```diff
@@ -28,4 +28,20 @@ export module AccountActions {
...
+      .exec()
+      .then((dto: any) => {
+        //TODO: Pending to set isOnboardingComplete flag
+        f(null, dto);
+      })
+      .catch((err: string) => {
+        f(err);
+      });
+  }
 }
```
Comment (whitehatdevv) [inline src/http/actions/Account.ts:47]: This action is not the appropriated one to get the photo by a current ID. here you are looking for an account but not for the photo indicated by the Id, please don't copy and paste the code I made before.

**Review by whitehatdevv — CHANGES_REQUESTED** [review]

File: src/http/actions/Account.ts (hunk 1 above)
Comment (octapf) [reply src/http/actions/Account.ts]: @whitehatdevv I have changed the action to get the photoId correctly

File: src/http/common/Base/static/Base.AccountEndpoints.ts [hunk 2]
```diff
@@ -4,6 +4,6 @@
  */
 export enum AccountEndpoints {
   Profile = '/profile',
-  PhotoId = '/profile/photo/:photoId',
+  PhotoId = '/account/:accountId/profile/photo/:photoId',
```
Comment (whitehatdevv) [inline src/http/common/Base/static/Base.AccountEndpoints.ts:7]: This route should be `/account/:accountId/photo/:photoId`. Please change it

**Review by whitehatdevv — CHANGES_REQUESTED** [review]

File: src/http/common/Base/static/Base.AccountEndpoints.ts (hunk 2 above)
Comment (octapf) [reply src/http/common/Base/static/Base.AccountEndpoints.ts:7]: @manu En notion está distinto
image

Comment (whitehatdevv) [reply src/http/common/Base/static/Base.AccountEndpoints.ts:7]: Then it's a mistake on notion, please change the route on your code and notion too

Comment (octapf) [reply src/http/common/Base/static/Base.AccountEndpoints.ts:7]: También el BEND-14 tiene profile/photo

Comment (whitehatdevv) [reply src/http/common/Base/static/Base.AccountEndpoints.ts:7]: Then change too, it was my fault, I'm sorry

Comment (octapf) [reply src/http/common/Base/static/Base.AccountEndpoints.ts:7]: Ok no problem!

Comment (octapf) [reply src/http/common/Base/static/Base.AccountEndpoints.ts:7]: Already fixed. Will appear on next commit.

**Pull request merged**